test/1.0/node_modules/serverless-build-plugin

```diff
@@ -1 +0,0 @@
-../../../dist
```

Comment:
What changed here ???

Reply:
I'm not sure, I couldn't revert it for some reason. I don't think it actually changes anything